tasks/eth/mode-001-MCP-L1/README.md

```diff
@@ -0,0 +1,35 @@
+# Mode Mainnet MCP L1 Upgrade
+
+Status: READY TO SIGN
+
+## Objective
+
+This is the playbook for executing Upgrade #1, Mode Mainnet Multi-Chain Prep (MCP) L1.
```

Comment:
I think we should still call this ugprade 6, since that's the name in the governance forum and the sepolia playbook. We don't really have a strong convention here though